examples/step-18/doc/intro.dox

```diff
@@ -596,6 +596,3 @@ addition, gravity acts on the body.
 The program text will reveal more about how to implement this situation, and
 the results section will show what displacement pattern comes out of this
 simulation.
-
-
```

Comment:
I think I remember that you need one empty line at the end of the file so that the next heading comes out alright. Can you double check?

Reply:
Hm. I remember this being the case somewhere similar but things look fine. Most of the other tutorial programs (e.g., step-1) have also had such blank lines removed so I think its okay.